client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Page Content section on the Publication setup page
![Screenshot showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/1e2f3881-c285-4be2-ac51-0f64b0b321a1)

Reply:
These now correctly appear as 16px.